openapi/components.yml

```diff
@@ -83,7 +83,17 @@ components:
       type: array
       items:
         type: string
-      example: ["Transport", "Culture", "Economy"]
+      example: [ "Transport", "Culture", "Economy" ]
+
+    ReportingPeriodRangeData:
+      type: object
+      properties:
+        start_date:
+          type: string
+          format: date-time
+        end_date:
+          type: string
+          format: date-time
```

Comment:
might be worth adding example so datetime format is clearly shown?

Reply:
will do